### PR TITLE
feature: add buildflag

### PR DESCRIPTION
**Pull request by blackholegalaxy**
Add support for Xcode buildFlag in the plugin

Closes: #37 

@Almouro

### CODE REVIEW

File: lib/fastlane/plugin/cordova/actions/cordova_action.rb [hunk 1]
```diff
@@ -28,17 +28,17 @@ def self.get_platform_args(params, args_map)
         args_map.each do |action_key, cli_param|
           param_value = params[action_key]
 
-          if action_key == 'build_flag' && param_value.kind_of?(Array)
+          if action_key.to_s == 'build_flag' && param_value.kind_of?(Array)
             unless param_value.empty?
               param_value.each do |flag|
-                platform_args << "--#{cli_param}=#{Shellwords.escape(flag)}"
+                platform_args << "--#{cli_param}=\"#{flag}\""
```
Comment (janpio) [inline lib/fastlane/plugin/cordova/actions/cordova_action.rb]: doesn't like the Shellwords.escape?

Comment (blackholegalaxy) [reply lib/fastlane/plugin/cordova/actions/cordova_action.rb]: I will test without it and improve later. Because it escaped the = and thus didn't work

Comment (blackholegalaxy) [reply lib/fastlane/plugin/cordova/actions/cordova_action.rb]: it's now reapplied

**Comment by Almouro** [comment]
Hi @blackholegalaxy, thanks for the PR! great feature! 🎉 

And @janpio, thanks for the review, do you want to become a collaborator on this repository? You're making awesome reviews!

**Comment by blackholegalaxy** [comment]
De rien :)

**Pull request merged**